src/Coalesce.Web.Tests/WebUiTests.cs

```diff
@@ -59,7 +59,7 @@ private void RunUiTest(IWebDriver driver)
 
             IWebElement personLink = driver.FindElement(By.PartialLinkText("Person"));
             personLink.Click();
-            driver.Manage().Timeouts().ImplicitlyWait(TimeSpan.FromSeconds(10));
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
```

Comment:
Should this be included here? Does not appear to be related to nuget stuff.
Same question for all of the code changes below too.